localdev/starters/example-starter/ocw-studio.yml

```diff
@@ -0,0 +1,6 @@
+collections:
+  - label: "Page"
+    name: "page"
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Content", name: "content", widget: "markdown"}
```

Comment:
We should remove this line as every markdown file is assumed to have markdown formatted content, and every page will be represented by a markdown file.